plasmapy/atomic/tests/test_parsing.py

```diff
@@ -255,7 +255,7 @@ def test_parse_InvalidParticleErrors(arg, kwargs):
         _parse_and_check_atomic_input(arg, **kwargs)
 
 
-@pytest.mark.parametrize('arg', _special_particles)
+@pytest.mark.parametrize('arg', ParticleZoo.everything - {'p+'})
```

Comment:
:clap: 

:clap: 

:clap: 

Beautiful.

Did you seriously just go to all that effort to be able to write "everything but protons" in elegant Python?

That's amazing. Well done! This IS really elegant Python. :laughing:

Reply:
I also did this because I think users studying cosmic rays would end up finding it useful to be able to easily access the sets of leptons, fermions, etc.  This also cleaned up the code in a bunch of different places since we no longer have to worry about a zoo of separate private variables like `_leptons`, `_fermions`, etc..  I should have used a class for this from the beginning!

Reply:
That is, of course, very reasonable. :+1: